### PR TITLE
User story 2

### LINKED ISSUE #7: User Story 2, Visitor Navigation
```
As a visitor
I see a navigation bar
This navigation bar includes links for the following:
- a link to return to the welcome / home page of the application ("/")
- a link to browse all books ("/books")
```

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [x] edge case ("sad path") tests written
- [x] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #31

child of #33

### PR DESCRIPTION
**Pull request by chakeresa**
This PR/branch...
 - Resolves #7 (User Story 2 - navigation bar)
 - Adds a navigation bar (`nav` element) to `application.html.erb` with links to the homepage and the books index
 - Adds a welcome page
 - Uses `ActionView::Helper`s in all the views
 - Moves `flash[:notice]`s to `application.html.erb`
 - Adds some `flash[:notice]`s for alert upon successful resource creation/destruction
 - Tests all of the above
 - Fixes the bug of no reload after destroying an author :-)
 - Adds `BC` to view when `year_published` < 0

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,6 +9,17 @@
...
 
   <body>
+    <% if !@hide_nav %>
+      <nav>
+        <%= link_to "Home", root_path %>
+        <%= link_to "Browse All Books", books_path %>
+      </nav>
+    <% end %>
+
+    <% if flash[:notice] %>
```
Comment (DavisC0801) [inline app/views/layouts/application.html.erb:19]: Loving this refactor. Very good.

**Review by DavisC0801 — APPROVED** [review]
Sorry that there isn't a lot to say on these, really solid work and good tests.
